plugins/woocommerce/src/Blocks/BlockTypes/FeaturedCategory.php

```diff
@@ -82,19 +83,25 @@ protected function get_item_image( $category, $size = 'full' ) {
 	 * @return string
 	 */
 	protected function render_attributes( $category, $attributes ) {
-		$title = sprintf(
-			'<h2 class="wc-block-featured-category__title">%s</h2>',
-			wp_kses_post( $category->name )
-		);
+		$output = '';
 
-		$desc_str = sprintf(
-			'<div class="wc-block-featured-category__description">%s</div>',
-			wc_format_content( wp_kses_post( $category->description ) )
-		);
+		// Backwards compatibility: Only render legacy attributes if `editMode` exists as boolean value
+		// This allows us to distinguish between old and new version of the block (which accept inner blocks).
+		if ( array_key_exists( 'editMode', $attributes ) && is_bool( $attributes['editMode'] ) ) {
+			$legacy_title = sprintf(
+				'<h2 class="wc-block-featured-category__title">%s</h2>',
+				wp_kses_post( $category->name )
+			);
+
+			$output .= $legacy_title;
 
-		$output = $title;
-		if ( $attributes['showDesc'] ) {
-			$output .= $desc_str;
+			if ( array_key_exists( 'showDesc', $attributes ) && $attributes['showDesc'] ) {
```

Comment:
This should be like this:

```suggestion
			if (
				! isset( $attributes['showDesc'] ) ||
				( isset( $attributes['showDesc'] ) && false !== $attributes['showDesc'] )
			) {
```

Otherwise, updating the _Featured Category_ block doesn't show the description by default.